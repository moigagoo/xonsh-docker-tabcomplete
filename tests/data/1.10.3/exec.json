{
        "arguments": [
                "-d", "--detach",
                "--detach-keys",
                "--help",
                "-i", "--interactive",
                "--privileged",
                "-t", "--tty",
                "-u", "--user"
        ],
        "commands": []
}
