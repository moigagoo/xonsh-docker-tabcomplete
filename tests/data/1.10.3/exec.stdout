
Usage:	docker exec [OPTIONS] CONTAINER COMMAND [ARG...]

Run a command in a running container

  -d, --detach         Detached mode: run command in the background
  --detach-keys        Override the key sequence for detaching a container
  --help               Print usage
  -i, --interactive    Keep STDIN open even if not attached
  --privileged         Give extended privileges to the command
  -t, --tty            Allocate a pseudo-TTY
  -u, --user           Username or UID (format: <name|uid>[:<group|gid>])
